document_page/models/document_page_history.py

```diff
@@ -19,6 +19,14 @@ class DocumentPageHistory(models.Model):
     content = fields.Text()
     diff = fields.Text(compute='_compute_diff')
 
+    company_id = fields.Many2one(
```

Comment:
I think this one should be related to the page. It's a nonsense to have a revision under one company and the main page under other, don't you think?

Reply:
It is related to the page

Reply:
Sorry, I didn't see it! Any way, it would be good to be stored for avoiding penalty when record rule is evaluated.

Reply:
I agree! Added index aswell